SIT.Manager.Avalonia/ViewModels/SettingsPageViewModel.cs

```diff
@@ -107,7 +122,8 @@ public partial class SettingsPageViewModel : ViewModelBase
 
     protected override void OnPropertyChanged(PropertyChangedEventArgs e) {
         base.OnPropertyChanged(e);
-
+        _localizationService.Translate(CurrentLocalization);
```

Comment:
This will call the localization code on every property change regardless of if the localization was actually changed

Reply:
Sure, sorry, I will add propertychange based on if the Settings current localization was changed.